src/core.js

```diff
@@ -696,17 +696,8 @@ define([
             $modal = $(modal_content({
                 title: title,
                 closeButtonTitle: closeButtonTitle
-            })),
-            $body = $("body"),
-            keyup = function (event) {
-                if (event.keyCode === 27) { // escape
-                    $modal.modal('hide');
-                }
-            };
-        $body.on("keyup", keyup);
-        $modal.on('hide', function () {
-            $body.off("keyup", keyup);
```

Comment:
That's how bootstrap modals behave by default (the `keyboard` parameter defaults to true) and having this extra handler prevented me from turning off that behavior.